recipes-connectivity/iproute2/iproute2_4.%.bbappend

```diff
@@ -1,10 +1,7 @@
-PR .= ".1"
+PR .= ".2"
 
 PACKAGE_ARCH = "${MACHINE_ARCH}"
-EXTRA_OEMAKE += "-j 1"
```

Comment:
Are you sure this is ok to remove? Is it documented that they used to have trouble building in parallel and that they don't anymore?